internal/gps/internal/paths/paths.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2017 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package paths
+
+import "strings"
+
+// This was lovingly lifted from src/cmd/go/pkg.go in Go's code.
```

Comment:
The provenance is actually pretty important here, but yes, it could also use a line describing its purpose:

`IsStandardImportPath indicates whether the provided path should be considered part of the standard library.`

That should work as an opening line, then followed by the note about where it comes from.